### ISSUE TITLE
A way to apply manual changes via changing the json files?

### ISSUE THREAD

**Issue by Rewyn** [RPG-Maker-Translator/RPG-Maker-Translator#9]
Not sure if there's already a way to do this, but I just tried out this nice little tool, and it did work mostly fine. Except right at the second line of dialogue, it did translate the name of a picture in the event, but not in the pictures folder itself. So now it's looking for a picture with a name that doesn't exist.
I'd be more than happy to manually change the line back to the old one with the japanese name (which wasn't hard to find). 
While it's entirely possible to do this via RPG Maker itself, it also adds a couple more steps and also requires you to search for the correct map each time, which can be somewhat of a hassle with hundreds of maps. By doing it just via json files, I can just open all of them in notepad++ and simply search for the name of the picture (or whatever else gives me trouble) and change it back to the original.

**Comment by RenegadeBeam** [comment]
After some searching via google I found this github-project. By the looks of it, it could solve your problem. 
Just change the lines in the respective Json files and convert the files by using this script to rvdata.
Then you need to overwrite the existing rvdata with your new one. 
https://github.com/DICE2000/rvdata2json

Quote: "convert json to rvdata"

**Comment by Rewyn** [comment]
Yup, sounds exactly what I'm looking for. I have to admit, I didn't even get the idea someone else might've done something like that, for some reason... So thanks for looking it up! :)
